### ISSUE TITLE
Different shading for odd/even octaves to separate octave ranges

### ISSUE THREAD

**Issue by mikobuntu** [LMMS/lmms#669]
Hi , I have just noticed a tracker called "Sunvox" , which has a piano_roll type of keyboard setup, I didnt look much at how it works, but it was the GUI element that struck me as being clever.
Each octave has the white keys in order light.. darker .. light etc, this for me would be really helpful for a novice to quickly help them to recognise where each octave starts/ends. Both sets of key colours white and light grey both have images set to be darker in accordance to their hue.

check out this video of Sunvox keyboard in action 

https://www.youtube.com/watch?v=uajwIouNb6I

thanks Mikobuntu

**Comment by Hussam-Eddin-Alhomsi** [comment]
I don't know about the Piano Roll back in 2014, but the Piano Roll now is easy in terms of octave recognition.
I don't think there is any need for what is proposed by @mikobuntu.
This is the current Piano Roll:
![image](https://user-images.githubusercontent.com/25503477/28996590-7c228094-7a0c-11e7-96da-c552d03c741a.png)

**Comment by mikobuntu** [comment]
I still stand by my original request of using a color element to quickly separate ( visually ) the octave ranges in Piano_roll .
Here's a rotated view of Sunvox Piano_roll keys just for a reference to those that can't be bothered to watch the original video above.
![sunvoxkeys](https://user-images.githubusercontent.com/335217/28996882-46662494-7a00-11e7-8d3d-f5e188051b30.jpg)

**Comment by Hussam-Eddin-Alhomsi** [comment]
I still don't think that there is a need for color shading octaves, because you can see in the image in my previous comment that C1 and C2 are in black and the other notes are in green, and those other notes can also be configured via `Enable note labels in piano roll`. So I think there is enough distinction between octaves and there is no need for more.

If some users still need this feature for more distinction, it should be user configurable, just like `Enable note labels in piano roll`.

**Comment by mikobuntu** [comment]
@Sawuare Yes I agree that if it were to be implemented , it should be user configurable.
